docs/resource/process.md

```diff
@@ -156,13 +156,19 @@ Examples for some JavaScript runtimes
 
 ### .NET Runtimes
 
-TODO(<https://github.com/open-telemetry/opentelemetry-dotnet/issues/1281>): Confirm the contents here
+.NET instrumentation should fill in the values by following values:
 
-| Value | Description |
-| --- | --- |
-| `dotnet-core` | .NET Core, .NET 5+ |
-| `dotnet-framework` | .NET Framework |
-| `mono` | Mono |
+- `process.runtime.name` - Fill in the value by the name of runtime.
+- `process.runtime.version` - Fill in the value of `System.Environment.Version`.
```

Comment:
For older versions, in the Elastic APM Agent we had some issues around using `System.Environment.Version`, findings are documented [here](https://github.com/elastic/apm-agent-dotnet/blob/main/src/Elastic.Apm/Helpers/PlatformDetection.cs#L116).

I think affected versions are already deprecated. I leave it up to you to decide if this is needed or not for OTel. Probably more relevant for the implementation and less for the SemConv spec.

Reply:
All OTel .NET repositories/SIGs agreed to support only supported versions of.NET Framework (4.6.2+) and .NET (6+).

I would skip details for implementation for edge cases (obsolete .NET runtimes).

I have changed typography for this (should -> SHOULD) for all languages: d6d65358e52aaa80411bb21b61e2a890842c9274. It ~~should~~ SHOULD be more visible.